dvc/command/config.py

```diff
@@ -1,100 +1,59 @@
 from __future__ import unicode_literals
 
-import os
 import argparse
 import logging
 
 from dvc.command.base import CmdBaseNoRepo, append_doc_link
 from dvc.config import Config
-from dvc.exceptions import DvcException
 
 
 logger = logging.getLogger(__name__)
 
 
 class CmdConfig(CmdBaseNoRepo):
     def __init__(self, args):
-        from dvc.repo import Repo, NotDvcRepoError
-
         super(CmdConfig, self).__init__(args)
 
-        try:
-            dvc_dir = os.path.join(Repo.find_root(), Repo.DVC_DIR)
-            saved_exc = None
-        except NotDvcRepoError as exc:
-            dvc_dir = None
-            saved_exc = exc
-
-        self.config = Config(dvc_dir, validate=False)
-        if self.args.system:
-            self.configobj = self.config._system_config
-        elif self.args.glob:
-            self.configobj = self.config._global_config
-        elif self.args.local:
-            if dvc_dir is None:
-                raise saved_exc
-            self.configobj = self.config._local_config
-        else:
-            if dvc_dir is None:
-                raise saved_exc
-            self.configobj = self.config._repo_config
-
-    def _unset(self, section, opt=None, configobj=None):
-        if configobj is None:
-            configobj = self.configobj
-
-        try:
-            self.config.unset(configobj, section, opt)
-            self.config.save(configobj)
-        except DvcException:
-            logger.exception("failed to unset '{}'".format(self.args.name))
-            return 1
-        return 0
-
-    def _show(self, section, opt):
-        try:
-            self.config.show(self.configobj, section, opt)
-        except DvcException:
-            logger.exception("failed to show '{}'".format(self.args.name))
-            return 1
-        return 0
-
-    def _set(self, section, opt, value):
-        try:
-            self.config.set(self.configobj, section, opt, value)
-            self.config.save(self.configobj)
-        except DvcException:
-            logger.exception(
-                "failed to set '{}.{}' to '{}'".format(section, opt, value)
-            )
-            return 1
-        return 0
+        self.config = Config(validate=False)
 
     def run(self):
         section, opt = self.args.name.lower().strip().split(".", 1)
 
         if self.args.unset:
-            return self._unset(section, opt)
+            self.config.unset(section, opt, level=self.args.level)
         elif self.args.value is None:
-            return self._show(section, opt)
+            logger.info(self.config.get(section, opt, level=self.args.level))
         else:
-            return self._set(section, opt, self.args.value)
+            self.config.set(
+                section, opt, self.args.value, level=self.args.level
+            )
+
+        return 0
 
 
 parent_config_parser = argparse.ArgumentParser(add_help=False)
```

Comment:
Just a quick reminder that `dvc pkg` still has the `-h` option disabled.

Reply:
So will it stay that way? :B sorry to insist just confused as to whether I should add it to the docs or nay.

Reply:
@jorgeorpinel It is enabled in https://github.com/iterative/dvc/pull/2012/commits/2116b4752fbf896f9368aaba892f542c88397472 . The code that you are pointing to is not related to pkg being hidden, it is just a helper.

Reply:
Oops, didn't add the help itself. Sorry. Thanks for a heads up! Fixed now :slightly_smiling_face:

Reply:
Thanks! I see it's enabled now 👽